liquidator/liquidator.js

```diff
@@ -285,7 +287,29 @@ class Liquidator {
       // Send the transaction or report failure.
       let receipt;
       try {
-        receipt = await liquidation.send(txnConfig);
+        // Make sure to keep trying with this nonce
+        const nonce = await this.web3.eth.getTransactionCount(this.account);
+
+        // Min Gas Price, with a max gasPrice of double
+        const minGasPrice = parseInt(this.gasEstimator.getCurrentFastPrice(), 10);
+        const maxGasPrice = 2 * minGasPrice;
```

Comment:
This is pretty awesome! however, I think we might want a more aggressive curve and `maxGasPrice`? The current `liquidationDeadline` is set to 5 mins and so under the current config the bot will only be able to bump by 10 GWEI 2 times before the tx fails due to the `liquidationDeadline`. I'd say if we want to use this mechanism we need to tune these parameters such that the bot tries more aggressively to ramp up gas prices within the 5 min window.

Reply:
We can also reduce the delay and make the function an exponential function if you please. I'm still not sure whats the best fit.

Reply:
Agree with. more aggressive curve. If possible to make it exponential then that'd work, otherwise I'd increase by at maybe even 50 to make sure transaction sends (and quickly!)